File: app/src/main/java/com/wealoha/social/beans/UserListGetData.java
package com.wealoha.social.beans;

import java.util.List;
import java.util.Map;

import com.wealoha.social.api.common.dto.ImageDTO;

public class UserListGetData extends ResultData {

	public List<UserDTO> list;
	public Map<String, ImageDTO> imageMap;
	public String nextCursorId;
	public boolean alohaGetLocked;
	public int alohaGetUnlockCount;

}

File: app/src/main/java/com/wealoha/social/beans/UserListGetData.kt
package com.wealoha.social.beans

data class UserListGetData(
    var list: List<UserDTO> = listOf(),
    var imageMap: Map<String, ImageCommonDto> = mapOf(),
    var nextCursorId: String? = null,
    var alohaGetLocked: Boolean = false,
    var alohaGetUnlockCount: Int = 0
) : ResultData() {

}